.goreleaser.yaml

```diff
@@ -12,6 +12,10 @@ builds:
     goarch:
       - amd64
       - arm64
+    ldflags:
+      - -X github.com/cnoe-io/idpbuilder/pkg/cmd/version.idpbuilderVersion={{ .Version }}
+      - -X github.com/cnoe-io/idpbuilder/pkg/cmd/version.gitCommit={{ .FullCommit }}
+      - -X github.com/cnoe-io/idpbuilder/pkg/cmd/version.buildDate={{ .CommitDate }}
```

Comment:
Using the [Name Templates](https://goreleaser.com/customization/templates/) provided in Goreleaser for the release flags, for consistency.